builtins/amp-ad.js

```diff
@@ -283,6 +283,10 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
+      if (!this.placeholder_) {
+        this.element.style.height = 0;
```

Comment:
@dvoytenko updated the code to have optional callbacks on change height - let me know if this is correct - ill update the tests and get this in.